### PR TITLE
Added try again screen, added shimmer effect and improved the response of fastag api

### PR DESCRIPTION
**Pull request by aman-netizen-ux**

### CODE REVIEW

**Comment by Anik-Chakraborty** [comment]
image
these try again button doesn't work at least it should show the loading effect and then show try again even no data is there

image
also the back button doesn't work in try again screen.

**Comment by Anik-Chakraborty** [comment]
for truck no CG07BC9186 the left timeline renders very _**strangely**_
image

**Review by Anik-Chakraborty — CHANGES_REQUESTED** [review]

**Review by Anik-Chakraborty — COMMENTED** [review]
Found this error in terminal while testing.... handle this exception because in production it might cause problem.
location from timeline : null
500
Error: NoSuchMethodError: '[]'
Dynamic call of null.
Receiver: null
Arguments: [0]
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 288:49      throw_
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 813:3   defaultNoSuchMethod
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 807:56  noSuchMethod
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 283:12  callNSM
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 294:25  _checkAndCall
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 431:10  callMethod
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 435:5   dsend
packages/shipper_app/functions/ongoingTrackUtils/FastTag.dart 25:52               getVehicleLocation
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 45:50                <fn>
dart-sdk/lib/async/zone.dart 1661:54                                              runUnary
dart-sdk/lib/async/future_impl.dart 147:18                                        handleValue
dart-sdk/lib/async/future_impl.dart 784:44                                        handleValueCallback
dart-sdk/lib/async/future_impl.dart 813:13                                        _propagateToListeners
dart-sdk/lib/async/future_impl.dart 584:5                                         [_completeWithValue]
dart-sdk/lib/async/future_impl.dart 657:7                                         callback
dart-sdk/lib/async/schedule_microtask.dart 40:11                                  _microtaskLoop
dart-sdk/lib/async/schedule_microtask.dart 49:5                                   _startMicrotaskLoop
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 177:15               <fn>

**Comment by Anik-Chakraborty** [comment]
image
getting this error when resizing the browser window from desktop to mobile or mobile screen size

Expected a value of type 'Set<Polyline>', but got one of type 'List<Set<Polyline>>'

The relevant error-causing widget was: 
  MobileMap MobileMap:file:///D:/shipperAppLiveasy/lib/screens/FastTagScreen.dart:296:14

File: lib/functions/ongoingTrackUtils/FastTag.dart [hunk 1]
```diff
@@ -21,21 +22,50 @@ class checkFastTag {
...
+        //Coming in the response.
+        String address = await fetchAddressForWeb(
+          double.parse(reversedList[i]['tollPlazaGeocode'].split(',')[0]),
+          double.parse(reversedList[i]['tollPlazaGeocode'].split(',')[1]),
+        );
+        reversedList[i]['address'] = address;
+      }
       return reversedList.cast<Map<String, dynamic>>();
     } else {
       debugPrint('API Error: ${response.statusCode} - ${response.body}');
```
Comment (Anik-Chakraborty) [inline lib/functions/ongoingTrackUtils/FastTag.dart:46]: Rather than printing the response body ... just debug print the error message only

Comment (aman-netizen-ux) [reply lib/functions/ongoingTrackUtils/FastTag.dart]: Resolved this issue.

**Review by Anik-Chakraborty — APPROVED** [review]

**Review by liveasy97 — APPROVED** [review]

**Pull request merged**